### PR TITLE
Adding Playground popup to home page

### LINKED ISSUE #4276: Add Meshery Playground popup on Layer5's home page.
A popup is required on the home page of the website (https://layer5.io/) that should redirect users to the Meshery Playground (https://play.meshery.io/). 

Some assests/ graphics that can be used for the poup are available here: https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?type=design&node-id=10869-15222

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Shivam-AfA**
**Description**

This PR fixes #4276 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Shivam-AfA** [comment]
@leecalcote This PR adds the playground popup on the home page. The popup is displayed after 2 seconds when the page loads. Users will be able to see the popup only once. Currently I have used an SVG created by @ritiksaxena124 for the content of the popup.

Here's how the popup looks like:

image

**Comment by ritiksaxena124** [comment]
just a suggestion, we can increase the opacity of the black overlay so that the pop is in more focus, something like [this](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?type=design&node-id=14019%3A37063&t=3J4FtMWggxoj7Ckv-1)

**Comment by Shivam-AfA** [comment]
Sure. I'll also add the drop shadow around the popup.

**Comment by l5io** [comment]
🚀 Preview for commit 9117540c20491345587029d5e3df666743d32daf at: https://646f230e30dd6c20da5cc767--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 95ca403339b9063ceb057dcb7cd8f726a7e57d26 at: https://646f8e805186751f6f9cd2ac--layer5.netlify.app

**Comment by leecalcote** [comment]
@Shivam-AfA @ritiksaxena124 Great work. Suggestions:

1) allow users to dismiss by clicking anywhere other than the modal.
2) make the entire modal / popup clickable in addition to the button.
3) keep this implementation, but also create a second implementation that we might use instead:
 --  change this to a popup in the lower, right hand corner that can be ignored if so desired and not a full-screen modal the forces the visitor to interact with it.

File: src/components/Playground-popup/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import PopupWrapper from "./popup.style";
+import popupImage from "./power-of-meshery.svg";
+import closeIcon from "./closeIcon.svg";
+import Button from "../../reusecore/Button";
+
```
Comment (Chadha93) [inline src/components/Playground-popup/index.js:6]: Let's wrap this in clickaway - https://www.npmjs.com/package/react-click-away-listener.

**Comment by leecalcote** [comment]
Do we really need another package for listening to a click away event? If so, that's fine.

**Comment by Shivam-AfA** [comment]
>Do we really need another package for listening to a click away event? If so, that's fine.
@Chadha93 @leecalcote I tried doing it without the package and was succesful. So there's no need for this now. I have added the suggested functionality.

**Comment by l5io** [comment]
🚀 Preview for commit 488bf296e8adb50437dcc00c8984aed4089b3e0a at: https://64734b367d4ce0316d31a295--layer5.netlify.app

**Comment by leecalcote** [comment]
@Shivam-AfA, let’s delay this popup until after 8 seconds. 

I mobile view, this popup still displays as a full-screen modal. The lower, right hand corner EOS what we’re looking for, though.

**Comment by Shivam-AfA** [comment]
@leecalcote I'll make another PR for a popup at the lower right hand corner of the screen. 

However, I have doubts if we should use that implementation for smaller screens, since it will be difficult to see such a small sized image.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks much. We don't want a full-screen modal, though. They are off-putting to our audience. 

In terms of mobile, yes, the lower right-hand corner is very much feasible. Use a different graphic / design for mobile.

**Comment by Shivam-AfA** [comment]
> Thanks much. We don't want a full-screen modal, though. They are off-putting to our audience. 
> 
> In terms of mobile, yes, the lower right-hand corner is very much feasible. Use a different graphic / design for mobile. 

Noted.

**Comment by leecalcote** [comment]
@Shivam-AfA yes, we'll need a different graphic for the small-sized mobile popup. // @ritiksaxena124

**Pull request closed without merging**